google_compute_engine_oslogin/utils/oslogin_utils.cc

```diff
@@ -171,14 +172,17 @@ bool HttpGet(const string& url, string* response, long* http_code) {
 
       code = curl_easy_perform(curl);
       if (code != CURLE_OK) {
+        curl_global_cleanup();
+        curl_easy_cleanup(curl);
```

Comment:
here and below, curl_easy_cleanup() should be before global_cleanup()

Reply:
and above :)

Reply:
Yep, thanks!